example/README.md

```diff
@@ -39,7 +39,7 @@ clientSecret = 'YOUR-SECRET-KEY'
 env = 'APPLICATION-ENVIRONMENT'
 ```
 
-Set the env to `production` or `sandbox`
+Set the env to `development` or `production` or `sandbox`
```

Comment:
remove the development environment

Reply:
@chiragphyllo what about we have defined in config.js ?